helm-charts/strimzi-kafka-operator/templates/020-RoleBinding-strimzi-cluster-operator.yaml

```
@@ -2,9 +2,13 @@
 {{- range append .Values.watchNamespaces .Release.Namespace }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+{{- if $root.Values.watchAnyNamespace }}
+kind: ClusterRoleBinding
+{{- else }}
 kind: RoleBinding
+{{- end }}
 metadata:
-  name: strimzi-cluster-operator
+  name: strimzi-cluster-operator-namespaced
```

Could you rename this only when the `watchAnyNamespace` is set? The way, it will keep the default names when installed only into a single namespace. I guess it might be something like this:

```suggestion
  {{- if $root.Values.watchAnyNamespace }}
  name: strimzi-cluster-operator-namespaced
  {{- else }}
  name: strimzi-cluster-operator
  {{- end }}
```

That way it will stay more compatible with all previous installations installed without it.